.github/scripts/run-tests.sh

```diff
@@ -89,6 +89,10 @@ case $1 in
         (cd $LOCAL_CHIPYARD_DIR/generators/mempress/software/src && make)
         run_binary BINARY=$LOCAL_CHIPYARD_DIR/generators/mempress/software/src/mempress-rocc.riscv
         ;;
+    chipyard-compressacc)
+        (cd $LOCAL_CHIPYARD_DIR/generators/compress-acc/software-zstd/compress && ./build-hcb-single-file.sh)
+        run_binary BINARY=$LOCAL_CHIPYARD_DIR/generators/compress-acc/software-zstd/compress/009987_cl0_ws12.riscv
```

Comment:
Use LOADMEM
![image](https://github.com/ucb-bar/chipyard/assets/14086183/3d7a3dd2-69b4-4166-b60a-f7df27cd0831)

Reply:
<img width="431" alt="Screenshot 2024-06-21 at 4 17 43 PM" src="https://github.com/ucb-bar/chipyard/assets/55655504/fc61e5ad-6eb3-4565-a9d3-4bd6b1782035">

Reply:
I think it is using loadmem

Reply:
`run-binary-fast` does not use LOADMEM.. it just disables printfs to avoid getting stuck on IO.